src/test/scala/com.lightbend.paradox/apidoc/MarkdownBaseSpec.scala

```diff
@@ -83,40 +91,41 @@ abstract class MarkdownBaseSpec extends FlatSpec with Matchers {
     writerContext(location).copy(properties = properties.toMap)
   }
 
-  def writerContext(location: Location[Page]): Writer.Context = {
+  def writerContext(location: Location[Page]): Writer.Context =
     Writer.Context(
       location,
       Page.allPaths(List(location.root.tree)).toSet,
+      reader = new Reader,
+      writer = new Writer,
       groups = Map("Language" -> Seq("Scala", "Java"))
     )
-  }
 
-  def pages(mappings: (String, String)*): Forest[Page] = {
+  def pages(mappings: (String, String)*): Forest[Page] =
+    pagesWithProperties(Map.empty[String, String], mappings: _*)
+
+  def pagesWithProperties(properties: Map[String, String], mappings: (String, String)*): Forest[Page] = {
```

Comment:
I added this so that with the upgrade, if one wanted to pass in properties which is now possible, they can. You can not have the properties map passed into the existing pages - varargs ...